### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by kabirwalia8300** [meredithdobrzynski/google_play_project#17]
The project dealt with understanding the importance of different features like pricing, categorization, etc on the ratings mobile applications receive on the Google Play Store. They identify ratings (and not downloads) as their target feature and have a mix of various data types in their dataset - containing information on ~10K apps.

The project did a great job to include concepts from class. Very clear and well explained in the reasoning behind different decisions. I like that the group experimented with different loss functions and also applied low rank models to deal with missing data. I further appreciate the analysis of train/test errors at each stage of the modeling and even discussion about potential noise. The visualizations were consistent and relevant too.

In terms of areas of improvement, I would like to have seen better error analysis and interpretation of model outcomes. This is important as the group wants to identify the strong, impactful features. The group reverts back to several of the correlations it identified between different features and Ratings but these correlations are linear. Considering that decision tree can be explored (as the group mentions that they do), it would have been nice to understand what those features were (sort of undoing the one-hot). Furthermore, the group talks about overfitting and underfitting in a general sense. It would have been nice to understand what cares were taken for different models (Random forests tend to overfit more than decision trees for example). And while very common regression metrics, the MSE/MAE aren't extremely interpretable from a business/user perspective. I would've also like to see sentiment analysis on the reviews. Looking at the offsets for specific examples and delving deeper into the analysis would have been very insightful. Nonetheless, given the constraints, a very well writting project!
